ts/features/itwallet/navigation/routes.ts

```diff
@@ -9,6 +9,9 @@ export const ITW_ROUTES = {
   IDENTIFICATION: {
     MODE_SELECTION: "ITW_IDENTIFICATION_MODE_SELECTION",
     IDP_SELECTION: "ITW_IDENTIFICATION_IDP_SELECTION",
+    SPID: {
+      LOGIN: "ITW_SPID_LOGIN"
+    },
```

Comment:
For consistency:
```suggestion
    SPID_LOGIN: "ITW_IDENTIFICATION_SPID_LOGIN"
```
What do you think?